tests/foreman/api/test_remoteexecution.py

```diff
@@ -190,6 +190,7 @@ def test_negative_time_to_pickup(
     global_ttp = module_target_sat.api.Setting().search(
         query={'search': 'name="remote_execution_time_to_pickup"'}
     )[0]
+    default_global_ttp = global_ttp.value
```

Comment:
For restoring the setting, you should use `setting_update` fixture!

Reply:
I can't do it in a fixture as I need it to be default for the first part of the test. Anyway, I decided to have this part separated to a test of its own which should solve the problem

Reply:
He can not use this fixture as he does not want the setting to be applied at the time when the first Job Invocation is started. Therefore the `setting_update fixture can not be used in this test.

Reply:
This test is so big and I didnt realize that the update setting is not needed in first job as u said!